labkey-client-api/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # The LabKey Remote API Library for Java - Change Log
 
-## version 4.0.0
+## version 4.0.0-jsonUpgrade-SNAPSHOT
```

Comment:
Is this intentional?

Reply:
Sorry, meant to make this a draft. Waiting on results from TeamCity. I'll re-request review once TeamCity has taken its pass.